include/Util/helper.php

```diff
@@ -112,6 +122,144 @@ function checked_array( $values, $current ) {
+ * @since 2.4.0
+ *
+ * @return string
+ */
+function get_email_failed_svg() {
+	return <<<EOT
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg class="el_sent_status--failed" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="15px" height="15px" viewBox="0 0 510 510" style="enable-background:new 0 0 510 510;" xml:space="preserve">
+<g>
+	<g id="do-not-disturb">
+		<path d="M255,0C114.75,0,0,114.75,0,255s114.75,255,255,255s255-114.75,255-255S395.25,0,255,0z M51,255c0-112.2,91.8-204,204-204
+			c45.9,0,89.25,15.3,124.95,43.35l-285.6,285.6C66.3,344.25,51,300.9,51,255z M255,459c-45.9,0-89.25-15.3-124.95-43.35
+			L415.65,130.05C443.7,165.75,459,209.1,459,255C459,367.2,367.2,459,255,459z"/>
+	</g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
+EOT;
+}
+
+/**
+ * Returns the Email sent SVG.
+ *
+ * @see   https://www.flaticon.com/free-icon/tick-inside-circle_61222
+ * @since 2.4.0
+ *
+ * @return string
+ */
+function get_email_sent_svg() {
+	return <<<EOT
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg class="el_sent_status--sent" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="15px" height="15px" viewBox="0 0 510 510" style="enable-background:new 0 0 510 510;" xml:space="preserve">
+<g>
+	<g id="check-circle-outline">
+		<path d="M150.45,206.55l-35.7,35.7L229.5,357l255-255l-35.7-35.7L229.5,285.6L150.45,206.55z M459,255c0,112.2-91.8,204-204,204
+			S51,367.2,51,255S142.8,51,255,51c20.4,0,38.25,2.55,56.1,7.65l40.801-40.8C321.3,7.65,288.15,0,255,0C114.75,0,0,114.75,0,255
+			s114.75,255,255,255s255-114.75,255-255H459z"/>
+	</g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
+
+EOT;
+
+}
+
+/**
+ * Gets the log row class by result code.
+ *
+ * @param int $result Mail sent status.
+ *
+ * @return string
+ */
+function get_log_row_class_by_result_code( $result ) {
+	$log_row_classes = array(
+		0 => 'el_email_sent_status--failed',
+		1 => 'el_email_sent_status--sent',
+	);
+	if ( empty( $result ) ) {
+		return $log_row_classes[0];
+	}
+
+	$result = absint( $result );
+	if ( array_key_exists( $result, $log_row_classes ) ) {
+		return $log_row_classes[ $result ];
+	}
+
+	return $log_row_classes[0];
```

Comment:
This entire function could be simplified to just

```suggestion
		if ( 1 === $result ) {
	    return 'el_email_sent_status--sent';
	}
	
	return 'el_email_sent_status--failed';
```